dev/benchmarks/macrobenchmarks/README.md

```diff
@@ -121,8 +121,8 @@ Example:
 ```sh
 cd dev/benchmarks/macrobenchmarks
 
-# Runs in profile mode using the CanvasKit renderer
-flutter run --web-renderer=canvaskit --profile -d web-server lib/web_benchmarks.dart
```

Comment:
Woops! Thanks for catching this, I didn't notice this usage of `web-renderer` when I grepped for it (was looking in code files, I guess!)